pycbc/inference/sampler/dynesty.py

```diff
@@ -80,12 +84,22 @@ def __init__(self, model, nlive, nprocesses=1,
         pool = choose_pool(mpi=use_mpi, processes=nprocesses)
         if pool is not None:
             pool.size = nprocesses
-
+        self.use_mpi = use_mpi
+        self.nprocesses = nprocesses
+        self.maxcall = maxcall
+        self.checkpoint_time_interval = checkpoint_time_interval
         self.run_kwds = {} if run_kwds is None else run_kwds
         self.nlive = nlive
         self.names = model.sampling_params
         self.ndim = len(model.sampling_params)
         self.checkpoint_file = None
+        self.__getstate__ = self.getstate
+        self.niter = niter
+        if ('maxcall' or 'maxiter') in self.run_kwds:
```

Comment:
I'm confused by this. So there will be no checkpointing by default, if you don't set maxcall in run_kwds? It seems self.maxcall isn't actually used anyway. Why the inconsistency?

Reply:
Shouldn't checkpointing just be enabled if checkpoint_time_interval is set?